### ISSUE TITLE
Broken Inference docstring

### ISSUE THREAD

**Issue by schmit** [blei-lab/edward#553]
The example that illustrates the Inference class returns a type error (at least for me): 
https://github.com/blei-lab/edward/blob/master/edward/inferences/inference.py#L36

```
from edward.models import Normal

mu = Normal(mu=tf.constant(0.0), sigma=tf.constant(1.0))
x = Normal(mu=tf.ones(50) * mu, sigma=tf.constant(1.0))

qmu_mu = tf.Variable(tf.random_normal([1]))
qmu_sigma = tf.nn.softplus(tf.Variable(tf.random_normal([1])))
qmu = Normal(mu=qmu_mu, sigma=qmu_sigma)

inference = ed.Inference({mu: qmu}, data={x: tf.zeros(50)})
```

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-98db5d314653> in <module>()
      9 qmu = Normal(mu=qmu_mu, sigma=qmu_sigma)
     10 
---> 11 inference = ed.Inference({mu: qmu}, data={x: tf.zeros(50)})

/usr/local/lib/python3.6/site-packages/edward/inferences/inference.py in __init__(self, latent_vars, data)
     80         raise TypeError("Latent variable value has an invalid type.")
     81       elif not key.get_shape().is_compatible_with(value.get_shape()):
---> 82         raise TypeError("Latent variable bindings do not have same shape.")
     83 
     84     self.latent_vars = latent_vars

TypeError: Latent variable bindings do not have same shape.
```

I'm just getting started with Edward and I couldn't figure out what's wrong.

**Comment by dustinvtran** [comment]
Oh, oops. That should be a scalar value inside `tf.Variable`. The error says that they have mismatched shapes:
```python
>>> qmu
<ed.RandomVariable 'Normal_2/' shape=(1,) dtype=float32>
>>> mu
<ed.RandomVariable 'Normal/' shape=() dtype=float32>
```

**Comment by dustinvtran** [comment]
Fixed in #558.
